lib/api/tweets/tweet_service.dart

```diff
@@ -341,6 +341,7 @@ class TweetService {
   }) {
     final params = <String, String>{}
       ..addParameter('trim_user', trimUser)
+      ..addParameter('count', count ?? 100)
```

Comment:
```suggestion
      ..addParameter('count', count)
```

We don't enforce defaults anywhere in the wrapper so we shouldn't enforce one here either.

(when removing the default, the change in the `tweet_service_test` can be discarded)

Reply:
Makes sense, I'll remove it from the test now. 👍🏾